include/mesh.h

```diff
@@ -298,6 +298,9 @@ class Mesh {
   //! Find cell neighbours
   void compute_cell_neighbours();
```

Comment:
Could we move the construction of `node_cell_maps_` to `compute_cell_neighbours` instead of a separate function.